cmd/util/cmd/common/flags.go

```diff
@@ -31,7 +31,7 @@ func InitWithDBFlags(cmd *cobra.Command) {
 	cmd.PersistentFlags().StringVar(&flagPebbleDir, "pebble-dir", DefaultPebbleDir,
 		"directory to the pebble dababase")
 
-	cmd.PersistentFlags().StringVar(&flagUseDB, "use-db", DefaultDB, "the database type to use, --badger or --pebble")
+	cmd.PersistentFlags().StringVar(&flagUseDB, "use-db", DefaultDB, "the database type to use, badger or pebble")
```

Comment:
Cobra has some features that might simplify this logic, like mutual exclusion.
```go
cmd.MarkFlagsMutuallyExclusive("datadir", "pebble-dir")
```

Then the command will error if the user ever provides both flags

Reply:
They are not mutually exclusive. You need to specify both datadir and pebble-dir, because DKG data is still stored in badger, and protocol data might be stored in badger or pebble. 

The `use-db` will choose which database to read protocol data from.

Reply:
Ah right. Good point 👍